src/btree/bt_walk.c

```diff
@@ -18,9 +18,16 @@ __ref_index_slot(WT_SESSION_IMPL *session,
 {
 	WT_PAGE_INDEX *pindex;
 	WT_REF **start, **stop, **p, **t;
+	uint64_t yield_cnt;
```

Comment:
This change uses a combination of `yield_cnt` and `yield_count` - I'd prefer if they were consistent, and I slightly prefer `yield_count`.